src/__tests__/rules.test.ts

```diff
@@ -2,7 +2,15 @@ import { existsSync } from 'fs';
 import { resolve } from 'path';
 import plugin from '../';
 
-const ruleNames = Object.keys(plugin.rules);
+const excludeRules = [
+  // require-tothrow-message has been renamed to require-to-throw-message, remove in major version bump
+  'require-tothrow-message',
+];
+
+const ruleNames = Object.keys(plugin.rules).filter(rule => {
```

Comment:
This could be short-handed

Reply:
I generally don't use implicit returns on arrow functions because I think they are harder to work with, but it doesn't matter to me either way. Do you want me to change it?

Reply:
I think I would, if you don't mind; we use shorthand arrow functions in most other places, and imo they can make filters easier to read by providing a clearer distinction between simple and advance filters.

For example, in this case the filter can just be `filter(rule => !excludeRules.includes(rules));`, which reads as "filter rules not included in excludeRules".

Any conditional more complex than this and I'd agree w/ you that it should be long form.

They tend to be nicer w/ TypeScript anyway, since you don't have to worry about returns being used by mistake if you short-hand.